Gemfile

```diff
@@ -34,7 +34,7 @@ group :development, :test do
   gem "nokogiri"
   gem "parser", "2.7.1.0"
   gem "pry-byebug"
-  gem "rspec-rails"
+  gem "rspec-rails", "~> 4.0.0.beta4"
```

Comment:
v4 has been released: https://github.com/rspec/rspec-rails/releases/tag/v4.0.0

Reply:
dropped this commit :+1: